kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/FinalizingEnqueueTests.kt

```diff
@@ -0,0 +1,12 @@
+package kotlinx.kover.gradle.plugin.test.functional.cases
+
+import kotlinx.kover.gradle.plugin.test.functional.framework.checker.CheckerContext
+import kotlinx.kover.gradle.plugin.test.functional.framework.starter.TemplateTest
+
+internal class FinalizingEnqueueTests {
+    @TemplateTest("android-subproject-apply", [":app:koverHtmlReportDebug"])
+    fun CheckerContext.testPluginsOrder() {
+        // if Kotlin Gradle Plugin is applied before any Android Gradle Plugin and Kover is applied from `subprojects` block, then `Attempt to queue after finalizing` error occurs
```

Comment:
```suggestion
        // if Kotlin Gradle Plugin is applied before any Android Gradle Plugin and Kover is applied from `subprojects` block, `Attempt to queue after finalizing` error should not occur
```

If you fixed it, I think it doesn't occur anymore?

Reply:
Yep )